redis/src/geo.rs

```diff
@@ -52,7 +52,7 @@ impl ToRedisArgs for Unit {
 ///
 /// * You may want to use either `f64` or `f32` if you want to perform mathematical operations.
 /// * To keep the raw value from Redis, use `String`.
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Eq)]
```

Comment:
IIRC this will prevent you from using floats as `T`, since floats don't derive `Eq`. If this is intentional, you might want to fix the documentation to not reference floats.

Reply:
Good call, fixed that up. Thanks for the review!